package.json

```diff
@@ -34,53 +23,45 @@
   ],
   "license": "MIT",
   "scripts": {
-    "lint": "eslint -c ./src/index.js \"./test/**/*\""
+    "lint": "eslint -c ./src/index.js \"./test/**/*\"",
+    "postinstall": "lerna bootstrap",
+    "lerna:version": "lerna version --no-private",
+    "lerna:publish": "lerna publish --no-private"
   },
+  "workspaces": [
+    "packages/*"
+  ],
   "devDependencies": {
     "@semantic-release/changelog": "^6.0.1",
     "@semantic-release/exec": "^6.0.3",
     "@semantic-release/git": "^10.0.1",
-    "@semantic-release/github": "^8.0.4",
+    "@semantic-release/github": "^8.0.6",
     "@semantic-release/npm": "^9.0.0",
-    "@types/jest": "^27.5.1",
-    "@types/react": "^18.0.14",
-    "dotenv-cli": "^5.1.0",
-    "eslint": "^8.18.0",
-    "jest": "^27.5.1",
-    "jest-extended": "^2.0.0",
-    "prettier": "^2.7.1",
-    "react": "^18.2.0",
-    "replace-json-property": "^1.8.0",
-    "semantic-release": "^19.0.3",
-    "ts-jest": "27.1.5",
-    "typescript": "^4.7.4"
-  },
-  "dependencies": {
-    "@react-native-community/eslint-config": "^3.0.3",
-    "@typescript-eslint/eslint-plugin": "^5.29.0",
-    "@typescript-eslint/parser": "^5.29.0",
+    "@types/eslint": "^8.4.6",
+    "@types/jest": "^29.1.2",
+    "@types/node": "^16.11.7",
+    "@types/react": "^18.0.21",
+    "dotenv-cli": "^6.0.0",
+    "esbuild": "^0.15.10",
+    "eslint": "^8.25.0",
     "eslint-config-prettier": "^8.5.0",
-    "eslint-import-resolver-typescript": "^2.7.1",
     "eslint-plugin-eslint-comments": "^3.2.0",
     "eslint-plugin-import": "^2.26.0",
-    "eslint-plugin-jest": "^26.5.3",
-    "eslint-plugin-jest-formatting": "^3.1.0",
-    "eslint-plugin-jsx-a11y": "^6.5.1",
+    "eslint-plugin-jsx-a11y": "^6.6.1",
     "eslint-plugin-no-secrets": "^0.8.9",
-    "eslint-plugin-prettier": "^4.1.0",
-    "eslint-plugin-react": "^7.30.0",
-    "eslint-plugin-react-hooks": "^4.5.0",
-    "eslint-plugin-react-native": "^4.0.0",
-    "eslint-plugin-react-native-a11y": "^3.2.1",
-    "eslint-plugin-sonarjs": "^0.13.0",
-    "eslint-plugin-unicorn": "^42.0.0",
-    "eslint-plugin-woke": "^1.0.1"
-  },
-  "peerDependencies": {
-    "eslint": "8.x",
-    "jest": "27",
-    "prettier": "2.x",
-    "react": "17 - 18",
-    "typescript": "4.x"
```

Comment:
why are the peer deps gone?

Reply:
The packages are in the `packages/*`-folder.
The monorepo itself has no use for peerDependencies.

Reply:
this makes a lot of sense